cryptofeed/feedhandler.py

```diff
@@ -205,18 +205,19 @@ def run(self, start_loop: bool = True, install_signal_handlers: bool = True):
             be called from the main/parent thread's event loop
         """
         if len(self.feeds) == 0:
-            LOG.error('No feeds specified')
-            raise ValueError("No feeds specified")
+            txt = f'FH: No feed specified. Please specify at least one feed among {list(_EXCHANGES.keys())}'
+            LOG.critical(txt)
+            raise ValueError(txt)
 
-        if start_loop:
```

Comment:
im not sure removing the check for start_loop here is a good idea. If the user is managing the loops themselves, they should enable uvloop if they want to, i dont think its a good thing to enable it after they've indicated they want to control the loop.

Reply:
Good point :+1: I have reverted this change and added a comment based on your comment.